src/modules/mod.rs

```diff
@@ -40,8 +41,16 @@ pub fn handle<'a>(module: &str, context: &'a Context) -> Option<Module<'a>> {
         "directory" => directory::module(context),
         "character" => character::module(context),
         "cmd_duration" => cmd_duration::module(context),
+        "conda" => conda::module(context),
         "dotnet" => dotnet::module(context),
         "env_var" => env_var::module(context),
+        "nodejs" => nodejs::module(context),
```

Comment:
Were these just all missing previously? :o

Reply:
I honestly don't know how those got in there, must've been something during a rebase

Reply:
I swear Git is just mocking me now...

Alright, if you don't mind, could you sort these into alphabetical order? Not entirely sure how they keep unsorting themselves, but we'd like to keep an order where we can (in accordance with the comment at the top of the `match`)